packages/@cdktf/hcl2cdk/lib/generation.ts

```diff
@@ -267,28 +289,28 @@ export function resource(
         }
       : undefined;
 
-  const expressions = [
-    ...asExpression(
-      scope,
-      resource,
-      key,
-      mappedConfig,
-      nodeIds,
-      false,
-      false,
-      getReference(graph, id) || overrideReference
-    ),
-  ];
+  const expressions = await asExpression(
+    scope,
+    resource,
+    key,
+    mappedConfig,
+    nodeIds,
+    false,
+    false,
+    getReference(graph, id) || overrideReference
+  );
```

Comment:
This does not clone the Array anymore, is that okay?

Reply:
Yeah, not sure why that even was a spread, I think it was left over from a time where we had more than one thing in this list by default ( I think I used to have ternary spreads in there `...(thisOverwrite ? ... : [])`